concepts/05 Widgets/Scheduler/030 Appointments/020 Add Appointments/030 Events.md

```diff
@@ -43,6 +43,73 @@ To execute certain commands before or after an appointment was added, handle the
         ],
         // ...
     })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            // ...
+            :on-appointment-adding="onAppointmentAdding"
```

Comment:
This is not how event handlers are specified in the markup. Please check with the Component Configuration Syntax article.